package.json

```diff
@@ -1,143 +1,99 @@
 {
   "name": "@jellyfish-dev/react-native-client-sdk",
-  "version": "0.1.6",
-  "description": "React native client sdk for the Jellyfish server.",
-  "main": "lib/commonjs/index",
-  "license": "Apache-2.0",
-  "module": "lib/module/index",
-  "types": "lib/typescript/src/index.d.ts",
-  "react-native": "src/index",
-  "source": "src/index",
-  "files": [
-    "src",
-    "lib",
-    "android",
-    "ios",
-    "cpp",
-    "*.podspec",
-    "!lib/typescript/example",
-    "!ios/build",
-    "!android/build",
-    "!android/gradle",
-    "!android/gradlew",
-    "!android/gradlew.bat",
-    "!android/local.properties",
-    "!**/__tests__",
-    "!**/__fixtures__",
-    "!**/__mocks__",
-    "!**/.*",
-    "!scripts/**.sh"
-  ],
+  "version": "7.4.1",
```

Comment:
Version should be changed. But not sure to what. Maybe 2.0.0. Or if we change repo name to 1.0.0 or 0.0.1 or sth else.

Reply:
I updated it to 0.3.0, as right now we have 0.1.6 version released